### DISCUSSION TITLE
Slow website start

### DISCUSSION THREAD

**Discussion by MaxDeavy** [linuxserver/Heimdall#1240] [Q&A]
Hi,

I have the problem that the initial connection to the Heimdall website always needs 2-3 seconds to load the page.
Couldn't find any solutions online, just a few people with the same problem.
Its only on Heimdall everything else (other services like pihole...) loads very fast.

Max

**Comment by thatcoleyouknow** [comment]
@MaxDeavy did you ever figure this out? I'm experiencing the same thing and I can't figure it out

**Reply by MaxDeavy** [reply]
No, I switched to another Dashboard.
First i used organizer but now I changed to "Homer" Dashboard, "Homarr" is also very nice.
